### PR TITLE
Wrap the sass variables in quotes.

### PR DESCRIPTION
**Pull request by brian-pearson**
Without the outer quotes, the sass processor was removing any inner double quotes.   This would give you css that was technically valid but susceptible to encoding issues.   With this change we preserve the double quotes in the final css

### CODE REVIEW

**Comment by brian-pearson** [comment]
@dbatiste can you take a look at this?   Here is what was happening:

The sass variable created would look like this:
$myvar: url("data:image/svg+xml,....");

Which looks cool .. but when sass processes that you get css like this:
url( data:image/svg+xml,....);

So it looks like sass drops the quotes.   Which normally is fine for most svg images .. but a recent image I tried had a close paren that mucked up the css.   And I think we want the quotes to be in the final css anyway to be more awesome

I'm not super sure the processing is the same for less .. I'll try to figure that out .. 

Let me know if this is dumb and I missed something :)

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "gulp-coveralls": "^0.1.3",
     "gulp-istanbul": "^0.3.1",
     "gulp-jasmine": "^1.0.1",
-    "gulp-jshint": "^1.8.6"
+    "gulp-jshint": "^1.8.6",
+    "jasmine-core": "^2.4.1"
```
Comment (brian-pearson) [inline package.json]: I got an error when I tried to run the tests, adding this guy got me through that.

**Comment by dbatiste** [comment]
Looks good!

**Comment by brian-pearson** [comment]
Thanks @dbatiste 

I don't have write access, you'll have to hook me up or else you can merge for me

**Comment by dbatiste** [comment]
Np.  I don't have access to give access.  :(

**Comment by dlockhart** [comment]
@brian-pearson should have access now

**Pull request merged**